nss-day-cohort-30/bangazon-site-suave-salmon
# Fixes

Fixes:
N/a
Changes proposed in this request & reasons behind organizational or architectural decisions:
-Search products by product in navbar
-search products by city on homepage

Steps to test:
-type l in navbar. Wheelbarrel should be in search results
-create a new product with nashville as the city 
-on home page search nashville. Your created product should appear.


System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-NA
Commit message:


Link to feature ticket:


@suave-salmon
